sanic/blueprint_group.py

```diff
@@ -130,6 +151,15 @@
         :param item: New `Blueprint` object.
         :return: None
         """
+        if self._version and item.version and self._version != item.version:
+            raise APIVersionMismatchException(
```

Comment:
I don't think this should be an exception. Maybe a warning, but probably not even that. We do not raise such mismatches when you drilldown Blueprint to handler. Indeed, it is a feature to allow the greater the specificity to win out.

Reply:
All righty. If not an exception, just better to ignore it completely and let the user decide how they want to manage it.